src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.storage;
+
+import core.basesyntax.model.Fruit;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private List<Fruit> fruits = new ArrayList<>();
```

Comment:
According to the checklist, you should use a data structure from the Collection framework that is suitable for your needs. A List may not be the best choice for representing a storage where quick search, insert, and update are needed. Consider using a Map with a fruit as the key and quantity as the value, which would be more efficient for these operations.